### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by orillusion-admin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 <body>
     <script type="module" src="/src/main.ts"></script>
+    <div>test</div>
```
Comment (lslzl3000) [inline index.html:11]: hi

Comment (lslzl3000) [reply index.html:11]: hh

File: index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
...
 </body>
 
-</html>
```
Comment (orillusion-admin) [inline index.html:13]: test review

File: index.html (hunk 1 above)
Comment (orillusion-admin) [reply index.html:11]: I have added one.

**Pull request merged**